lib/HLSL/DxilValidation.cpp

```diff
@@ -3510,8 +3595,9 @@ static void ValidateFunction(Function &F, ValidationContext &ValCtx) {
     // Shader functions should return void.
     if (isShader && !F.getReturnType()->isVoidTy())
       ValCtx.EmitFnFormatError(&F, ValidationRule::DeclShaderReturnVoid,
-                               {F.getName()});
+                               {F.getName()});    
 
+    // Validate parameter type.
```

Comment:
I think the move of this comment is left over from reverting an unrelated refactor here.  It's probably best to leave it alone in this PR to reduce unrelated, unnecessary churn.